specs/ClearBrowsingData.md

```diff
@@ -100,79 +100,82 @@ interface ICoreWebView2ClearBrowsingDataCompletedHandler;
 
 /// Specifies the datatype for the
 /// `ICoreWebView2Profile::ClearBrowsingData` method.
-[v1_enum] 
+[v1_enum]
 typedef enum COREWEBVIEW2_BROWSING_DATA_KINDS {
-  /// Specifies data stored by the AppCache DOM feature.
-  COREWEBVIEW2_BROWSING_DATA_KINDS_APP_CACHE = 1 << 0,
```

Comment:
We cannot remove a public enum value. Nor can we change the value of the other enum values. If we cannot support app cache anymore, we should leave the enum value, but update its documentation to note why its not supported, and what happens if you try to clear it (either nothing or return failure. Should pick 'nothing' in order to not break existing apps using this flag.)

Reply:
Yes, it should be. However, the enum of `APP_CACHE` has already been removed in the current source code. If we retain this, it would have conflict with the implementation.

Reply:
Oh I see. Please state that in a comment so others aren't confused by the change.

Reply:
Got it. I'll add this to the documentation.